apps/wing/src/commands/compile.ts

```diff
@@ -64,7 +53,7 @@ export async function compile(entrypoint: string, options: CompileOptions): Prom
         // file_id might be "" if the span is synthetic (see #2521)
         if (span !== null && span.file_id) {
           // `span` should only be null if source file couldn't be read etc.
-          const source = await fsPromise.readFile(span.file_id, "utf8");
+          const source = readFileSync(span.file_id, "utf8");
```

Comment:
Why this change? I think we should actually prefer to keep things async here as much as possible because this compiler is possibly used in other interactive settings where we don't want to block the main thread.

Reply:
The change is because there was a version that this was in a callback called from within WASM (a wasm import). But after (the rather complex) merge with Cristian's wingcompiler lib PR it bounced back into being called from TS, so I'll revert.